src/test/python/tempest_test.py

```diff
@@ -98,31 +98,57 @@ def expect_exception(body, exception_type):
 #expect_equal(sorted(twitter_2010_example.get_influencers("follows", 'alice', client)),
 #             sorted(['bob', 'carol']))
 
-test_mutation = False
+test_mutation = True
 # After running mutation tests, to avoid interference with tests above, recreate the test graph in docker by running the
 # example node/edge creation commands from the readme.
 # We've disabled these until we find a way to keep the mutated graph from changing results of future tests.
 if test_mutation:
     daniel = Node("user", "daniel")
-    client.add_node(daniel)
```

Comment:
Does it makes sense to keep some tests for `add_node`, `add_edge`, and `add_edges`?